src/styles/components/statusbar.less

```diff
@@ -0,0 +1,49 @@
+.status-bar {
+  display: inline-flex;
+  background-color: @gray;
+  margin-right: @base-spacing-unit * 2/3;
+  height: @base-spacing-unit * 1/3;
+  width: 114px;
+  max-width: 100%;
+
+  .bar {
+    background-color: currentColor;
+    border-left: 1px solid @black;
+    &:first-child {
+      border-left: none;
+    }
+  }
+}
+
+.status-bar-column {
+  @media (min-width: @screen-medium-max) {
+    width: 330px;
+  }
+  width: 138px;
+
+}
+.status-bar-wrapper{
+  display: inline-block;
+  .color-1 {
+    color: @white;
+  }
+  .color-2 {
+    color: @yellow;
+  }
+}
+
+.unknown {
```

Comment:
Those color definitions where for the dots since the bar has a to specific background colors so the background color was not overwritten I added a .dot block around those colors.